python/test/tests.py

```diff
@@ -6,7 +6,7 @@
 import sys
 import os
 import numpy as np
-from sklearn.cross_validation import train_test_split
+from sklearn.model_selection import train_test_split
```

Comment:
Does this mean we should also change the dependency version at https://github.com/yahoo/lopq/blob/master/python/setup.py#L86?

Reply:
Hey, model_selection was introduced in version 0.18, so I updated it accordingly. Sorry for such many commits, it's my first contribution to a public project and I'm still learning ;)